docs/UserGuide.md

```diff
@@ -70,11 +63,11 @@ Do have a look at our Table of Contents to locate the help you need!
    * **`clear`** : Deletes all contacts. You will find this handy when deleting the sample contacts stored on SociaLite.
```

Comment:
Should this be "**`delete`** `delete 3`:" instead? To standardise with the above pointers.

Reply:
Ooh! Excellent catch! I'll take note of this!

Reply:
Error amended in Line 61 of latest commit